SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -937,6 +937,22 @@ class CVariable {
    */
   inline virtual su2double GetMassFraction(unsigned long iPoint, unsigned long val_Species) const { return 0.0; }
 
+  /*!
+   * \brief A virtual member.
+   * \param[in] iPoint - Point index.
+   * \param[in] val_Species - Index of species s.
+   * \return Value of the mass fraction of species s.
+   */
+  inline virtual su2double GetEve(unsigned long iPoint) const { return 0.0; }
```

Comment:
Looks unused

Reply:
Yes, you're right, this was vestigial and isn't necessary. Removing it.